docs/changelog/5_x.rst

```diff
@@ -440,3 +440,102 @@ Bug fixes
 
 * Fix update rewrites on types that are children of updated type
   (:eql:gh:`#7073`)
+
+
+Beta 3
+======
+
+Additional changes
+------------------
+
+* Add ``cfg::Config._query_cache_mode``
+  (:eql:gh:`#7158`)
+
+Bug fixes
+---------
+
+* Make escaping strings more consistent.
+  (:eql:gh:`#7059`)
+
+* Separate triggers by kind in compilation.
+  (:eql:gh:`#7082`)
```

Comment:
For consistency — other items don't have a `.` at the end

```suggestion
* Make escaping strings more consistent
  (:eql:gh:`#7059`)

* Separate triggers by kind in compilation
  (:eql:gh:`#7082`)
```

Reply:
I've removed `.` from these logs, but we're not super-consistent about it overall in changelogs, it seems. Maybe we need to make a decision which is preferred with or without `.` and then do a quick update of all the changelogs?